presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -17,7 +17,8 @@ Java Runtime Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 * 64-bit required
-* version 11 recommended (higher versions are less tested)
+* version 11 recommended (lower versions not supported, higher versions less
```

Comment:
should we make it more clearly for some popular JDKs, e.g. for Oracle JDK, use 11.0.8+, for open JDK use 11.0.4+

Reply:
The next line mentions Azul Zulu as being the only JRE we actually run tests with .. beyond that we dont have reliable data and in the past we opted not to make a list of these JREs that might work. 

If we have any more specific info, maybe we should add it. I am not aware of reliable information along that though ..

Reply:
@mosabua i guess this is related to recent issue with oracle JDK (https://github.com/prestosql/presto/issues/4439), where we found out that 11.0.7 of AdoptOpenJDK works while Oracle JDK needs to be 11.0.8.

@ArvinZheng while i see a value in this particular piece of information, we do not have enough bandwith to maintain this information (we are not able to test on various JDK distributions available today). So I agree with Manfred, we should not be too detailed here, as this could be misudnerstood as we provide all the information, or all the supported versions precisely.  Saying "11" needs to suffice for now

Reply:
@mosabua @findepi understood.